nomad/structs/config/reporting.go

```diff
@@ -0,0 +1,80 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: BUSL-1.1
+
+package config
+
+import "github.com/hashicorp/nomad/helper/pointer"
+
+func DefaultReporting() *Reporting {
+	return &Reporting{
+		&LicenseConfig{},
+	}
+}
```

Comment:
Since this is about the `Reporting` struct it would be better if it were placed below, next to the other `Reporting` methods (or maybe pull the other methods up here).

Reply:
sure!